lib/gui/app/components/drive-selector/services/drive-selector.js

```diff
@@ -34,7 +34,8 @@ module.exports = function (ModalService, $q) {
    */
   this.open = () => {
     modal = ModalService.open({
-      template: './components/drive-selector/templates/drive-selector-modal.tpl.html',
+      name: 'drive selector',
```

Comment:
Purely for consistency purposes, perhaps this should be `name: 'drive-selector',` ?